pontoon/base/migrations/0009_change_pontoon_users_emails.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.13 on 2020-08-28 08:49
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def update_base_user_email(apps, schema_editor):
+    User = apps.get_model("auth", "User")
+    User.objects.filter(email="pontoon-sync@mozilla.com").update(email="pontoon-sync@example.com")
+    User.objects.filter(email="pontoon-gt@mozilla.com").update(email="pontoon-gt@example.com")
+    User.objects.filter(email="pontoon-mt@mozilla.com").update(email="pontoon-mt@example.com")
+
+def revert_base_user_email(apps, schema_editor):
+    User = apps.get_model("auth", "User")
+    User.objects.filter(email="pontoon-sync@example.com").update(email="pontoon-sync@mozilla.org")
+    User.objects.filter(email="pontoon-gt@example.com").update(email="pontoon-gt@mozilla.com")
+    User.objects.filter(email="pontoon-tm@example.com").update(email="pontoon-tm@mozilla.com")
+
+class Migration(migrations.Migration): 
+
+    initial = True
+
+    dependencies = [
+        ('base', '0008_add_systran_locales'),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            code=update_base_user_email, reverse_code=revert_base_user_email,
+        ),
+    ]
```

Comment:
Please add a [blank line at the end of file](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file).